node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
+    /**
+     * Fully-qualified name of the translator (required).
+     */
+    public readonly name: string;
+
+    /**
+     * List of interfaces implemented by the translator.
+     */
+    public readonly interfaceNames: string[];
+
+    /**
+     * Name of the onboarding module required by the translator.
+     */
+    public readonly onboardingName: string;
```

Comment:
Hmm, I'll go with `onboardingModuleName` for now, for lack of a better term. This reference might change anyway after we figure out the new design for onboarding.

---
In reply to: [71912279](https://github.com/openT2T/opent2t/pull/1#discussion_r71912279) [](ancestors = 71912279,71910673,71904172)